addon/components/ember-table-base-cell.js

```diff
@@ -39,7 +40,14 @@ export default class EmberTableCell extends Component {
 
   scheduleSync() {
     scheduler.schedule('sync', () => {
-      const { height } = this.element.getBoundingClientRect();
+      let { height } = this.element.getBoundingClientRect();
+
+      if (this.get('isFixed')) {
+        // Get the max height of the div element height and other cells height in case other cells
+        // are empty.
+        const fixedDivElement = this.element.getElementsByClassName('is-fixed')[0];
+        height = Math.max(height, fixedDivElement.getBoundingClientRect().height);
```

Comment:
@billy-addepar what happens if the user changes the row height or contents, and the row should shrink in height?